pkg/asset/agent/manifests/clusterdeployment.go

```diff
@@ -40,7 +40,7 @@ func (*ClusterDeployment) Dependencies() []asset.Asset {
 // Generate generates the ClusterDeployment manifest.
 func (i *ClusterDeployment) Generate(dependencies asset.Parents) error {
```

Comment:
instead of calling it just `i`, maybe change it to `cd`, an acronym for ClusterDeployment.